mailing/src/main/java/org/lukawska/trainSmart/mailing/domain/entities/MailEntity.java

```diff
@@ -0,0 +1,55 @@
+package org.lukawska.trainSmart.mailing.domain.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@Builder
+@Table(name = "mails")
+public class MailEntity {
```

Comment:
czy w Twoim projekcie ma sens uzycie protected constructora a klase pozostawic public? Ma to sens w pewnych przpyadkach ale chcialbmy uslyszec czy Twoj jest jednym z nich